src/main/java/school/lemon/changerequest/java/introduction/hw2/Task1.java

```diff
@@ -2,6 +2,7 @@
 
 public class Task1 {
     public static String makeTags(String tag, String text) {
-        return "";
+
+        return String.format("%s", "<" + tag + ">" + text + "</" + tag + ">");
```

Comment:
You should use correct pattern, there is no need to concatenate whole string.
There are no restrictions for constant values in pattern, for example: `<%s>` will produce `<argument_value>`